.travis.yml

```diff
@@ -6,6 +6,10 @@ matrix:
       python: "3.6"
     - os: linux
       python: "2.7"
+      addons:
+        apt:
+          packages:
+            - language-pack-de
```

Comment:
Just out of curiosity: what is `language-pack-de` good for and why is it only added in Python 2.7?

Reply:
Unfortunately the python 2.7 on travis and appveyor did not have the language installed, which is why the localized creation of german dates failed. I would not have cared much, but the using the locale package for parsing dates and numbers will be important for this package, so I decided to install it instead of changing the test cases

Reply:
Alright! Thanks for the explanation.